VideoLocker/src/main/java/org/edx/mobile/view/CourseOutlineActivity.java

```diff
@@ -50,6 +50,10 @@ protected void onLoadData() {
                 courseData.getCourse().getId(), courseComponentId);
         setTitle(courseComponent.getName());
 
+        // onLoadData is only called in case of viewing a section
```

Comment:
This statement is incorrect; `onLoadData()` is called for both course outline and subsection views.